### DISCUSSION TITLE
Suggestions / best practices for encrypting standalone SO install?

### DISCUSSION THREAD

**Discussion by oscar-o-oneill** [Security-Onion-Solutions/securityonion#6144] [Unsupported Versions] [answered]
Hi all! Loving Security Onion so far.

Just wondering if anyone has any tips regarding encryption and Security Onion. I just want to keep my data and packet captures private in case the device is lost when being transported or is stolen.

Any ideas here?

I was going to try out [this Full Disk Encryption method for Ubuntu 18.04](https://help.ubuntu.com/community/Full_Disk_Encryption_Howto_2019). Is this going to work OK or will it break anything?

Ideally it would be something like Windows Bitlocker and could use the TPM so I don't have to put in a password when the device turns on, but I am OK with typing the master password every time the machine boots to ensure it is secure.

If FDE isn't going to work, I would be OK with encrypting just a couple of key folders.

Or would you have any other ideas for keeping my data safe in case the Security Onion machine is lost?

Thanks a million!

**Comment by dougburks** [comment, marked as answer]
> I was going to try out [this Full Disk Encryption method for Ubuntu 18.04](https://help.ubuntu.com/community/Full_Disk_Encryption_Howto_2019).

Please feel free to try it out and let us know how it goes!

**Reply by oscar-o-oneill** [reply]
Hi @dougburks, thanks for your response. I'll be sure to try this out and report back.

Since I installed Security Onion via the standalone SO CentOS version, I will need to reinstall on Ubuntu to setup full disk encryption.

If you have any other tips or suggestions regarding encryption I am all ears!

Thank you.

P.S. Great work on Security Onion and cool to see you as the founder / CEO responding to my GitHub messages!

**Reply by oscar-o-oneill** [reply]
@dougburks, great news!

I got my install of Ubuntu 18.04.06 LTS installed, encrypted, and I got Security Onion up and running again.

The only hiccup I had when setting this all up was having my machine's IP address assigned by DHCP while it was being set up. I set a DHCP reservation, but for some reason the Security Onion installer kept failing.

So I just reinstalled Ubuntu and made sure the DHCP reservation was in place when I ran "so-network-setup". Setup went fine after I did that.

Now it's working great and I feel a lot safer knowing it's all encrypted.

Thanks again for your tip.

P.S. Also just bought a new 2TB SSD to use in this machine!
